pom.xml

```diff
@@ -5,20 +5,20 @@
     <modelVersion>4.0.0</modelVersion>
 
     <groupId>io.openliberty.sample</groupId>
-    <artifactId>jakarta-concurrency-demo</artifactId>
+    <artifactId>JakartaConcurrencyDemo</artifactId>
     <version>1.0-SNAPSHOT</version>
     <packaging>war</packaging>
 
     <properties>
-        <maven.compiler.source>21</maven.compiler.source>
-        <maven.compiler.target>21</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <!-- Plugins -->
         <version.liberty-maven-plugin>3.10.2</version.liberty-maven-plugin>
         <!-- OpenLiberty runtime -->
-        <version.openliberty-group>io.openliberty.beta</version.openliberty-group>
-        <version.openliberty-runtime>24.0.0.2-beta</version.openliberty-runtime>
-        <http.port>9082</http.port>
```

Comment:
This is switching to the latest GA version of Open Liberty, and the default 9080 port